src/history/repeat_commands/repeat_commands_specific.ts

```diff
@@ -0,0 +1,156 @@
+import { deepCopy, range, UuidGenerator } from "../../helpers";
+import { Getters } from "../../types";
+import {
+  AddColumnsRowsCommand,
+  AutoresizeColumnsCommand,
+  AutoresizeRowsCommand,
+  CoreCommand,
+  CreateChartCommand,
+  CreateFigureCommand,
+  CreateImageOverCommand,
+  CreateSheetCommand,
+  DeleteCellCommand,
+  DuplicateSheetCommand,
+  HideColumnsRowsCommand,
+  InsertCellCommand,
+  PasteCommand,
+  RemoveColumnsRowsCommand,
+  RepeatPasteCommand,
+  ResizeColumnsRowsCommand,
+  SortCommand,
+} from "./../../types/commands";
+import { transformSheetDependantCommand } from "./repeat_commands_generic";
+
+const uuidGenerator = new UuidGenerator();
+
+export function repeatCreateChartCommand(
```

Comment:
Probably another stupid question, but can't a chart also be a "target" dependant command ? If we select a zone and make a CREATE_CHART, then select another zone and CLTR+Y ?